### DISCUSSION TITLE
이제 graphl에서 item 목록 잡히진 않네요

### DISCUSSION THREAD

**Discussion by koouoo722** [changdoc/naver-vaccine-macro#92] [General]
크롬/Whale 사용해서 진행해보았습니다.

네이버 지도상 F12 graphl-response에서 item목록은 삭제된 상태여서
병원목록 생성은 어려운것 같은데.. 목록생성 해결방법이 있을까요?

병원별 링크확인 후 개별로 진행하는 방법 말고는
다른방법은 없는거겠죠..?

**Comment by changdoc** [comment]
지도에 마커로 대기중/ 마감/ 0 등 상태가 표시되었다면 graphql response가 정상적으로 있었다는걸 의미하는건데요. 여러번 새로고침을 해도 목록이 삭제된 상태일까요?

**Reply by koouoo722** [reply]
어제 진행하였을때에는 item목록이 없어서 당황했는데, 현재 진행하였을때 다시 item 목록이 나온것으로 보아 정상작동하는것 같습니다.
